.changeset/wicked-emus-deny.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/create-app': minor
+'@backstage/plugin-scaffolder-backend': minor
```

Comment:
Actually not sure that either of these are breaking right?

Reply:
Yeah should be patch